dask_ml/cluster/k_means.py

```diff
@@ -447,7 +447,10 @@ def init_scalable(
     # to do that.
 
     if len(centers) < n_clusters:
-        logger.warning("Found fewer than %d clusters in init.", n_clusters)
+        logger.warning(
```

Comment:
Can we keep the `%d` style formatting? It avoids the cost of formatting when the logger isn't enabled.

Reply:
You mean that the f-string syntax is slower?

Reply:
Can be, so I prefer to consistently use `%`-style for logging.